ebos/equil/initstateequil_impl.hh

```diff
@@ -694,9 +694,16 @@ void PhaseSaturations<MaterialLawManager, FluidSystem, Region, CellID>::deriveWa
         // cell.
         const auto pcow = this->press_.oil - this->press_.water;
 
-        sw = this->swatInit_.empty()
-            ? this->invertCapPress(pcow, this->waterPos(), isIncr)
-            : this->applySwatInit(pcow);
+        if (this->swatInit_.empty()) {
+            sw = this->invertCapPress(pcow, this->waterPos(), isIncr);
+        }
+        else {
+            auto [swout, newSwatInit] = this->applySwatInit(pcow);
+            if (newSwatInit == true)
```

Comment:
Any reason not to write this as
```C++
if (newSwatInit) {
    // ...
}
else {
    // ...
}
```
instead?  Comparing a `bool` directly to `true` is not something we usually do in this code base.